### PR TITLE
Feature/password strength meter

### PR DESCRIPTION
**Pull request by evalsocket**
## Proposed changes
Feature request in [ory/kratos](https://github.com/ory/kratos/issues/136) i.e. Add password strength meter. It can be used across the ory ecosystem 

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
- [x] I have read the [security policy](../security/policy)
- [x] I confirm that this pull request does not address a security vulnerability. If this pull request addresses a security
vulnerability, I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation within the code base (if appropriate)

### CODE REVIEW

File: passwordstrengthmeter/handler.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+//
+//
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: passwordStrength
+//       500: genericError
+func (h *Handler) PasswordStrength(rw http.ResponseWriter, r *http.Request, _ httprouter.Params) {
```
Comment (aeneasr) [inline passwordstrengthmeter/handler.go]: This function doesn't do anything, is that on purpose?

File: passwordstrengthmeter/handler.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	"github.com/nbutton23/zxcvbn-go"
+)
+
+const (
+	// PasswordStrengthPath is the path where you can check strength of password
+	PasswordStrengthPath = "/password/strength/meter"
+)
+
+// RoutesToObserve returns a string of all the available routes of this module.
+func RoutesToObserve() []string {
```
Comment (aeneasr) [inline passwordstrengthmeter/handler.go]: This is nowhere used. Please don't just copy source files from one destination to another and replace the names.

File: passwordstrengthmeter/handler_test.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/herodot"
+)
+
+func TestPasswordStrengthMeter(t *testing.T) {
+	alive := errors.New("not alive")
+	handler := &Handler{
+		H:             herodot.NewJSONWriter(nil),
+		VersionString: "test version",
```
Comment (aeneasr) [inline passwordstrengthmeter/handler_test.go]: This causes a compile error.

File: passwordstrengthmeter/handler_test.go [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+	"github.com/julienschmidt/httprouter"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/herodot"
+)
+
+func TestPasswordStrengthMeter(t *testing.T) {
+	alive := errors.New("not alive")
```
Comment (aeneasr) [inline passwordstrengthmeter/handler_test.go]: This is a copy of the health test, but we need a different test suite here.

File: passwordstrengthmeter/handler.go [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/ory/herodot"
+
+	"github.com/pkg/errors"
+
+	"github.com/ory/x/jsonx"
+
+	"github.com/nbutton23/zxcvbn-go"
```
Comment (aeneasr) [inline passwordstrengthmeter/handler.go]: This causes a compile error.

**Comment by evalsocket** [comment]
@aeneasr  do you want it in kratos or i put it in ory/x

**Comment by aeneasr** [comment]
@evalsocket this belongs into kratos :)

**Pull request closed without merging**